### DISCUSSION TITLE
GitHub academic discount coupon

### DISCUSSION THREAD

**Discussion by tunardev** [community/community#49979] [GitHub Education]
Hello, the system tells me: "Your GitHub academic discount coupon will expire in 11 days". When I say "Reverify your academic affiliation" I am redirected to https://education.github.com/discount_requests/application. And before I can say that I am still eligible for github education, when I write github education email, it says that this email is used for github education. What should I do?

**Comment by SwaggyMacro** [comment]
Same problem.

**Comment by erailea** [comment]
Same problem. Anyone was able to solve that issue?
